helpers.js

```diff
@@ -0,0 +1,114 @@
+const os = require("os");
+const childProcess = require("child_process");
+const fs = require("fs");
+const { normalize } = require("path");
+const compareVersion = require("compare-versions");
+const GitKey = require("./git-key");
+
+const homeDirectory = os.homedir();
+const isWin = os.platform() === "win32";
+
+async function verifyGitVersion() {
+  const minimalGitRequired = "2.10.0";
+  let gitVersion;
+
+  try {
+    const gitResult = await execCommand("git --version");
+    const gitVerNumArray = gitResult.split(" ")[2].trim().split(".");
+    gitVersion = `${gitVerNumArray[0]}.${gitVerNumArray[1]}.${gitVerNumArray[2]}`;
```

Comment:
How about `gitVerNumArray.slice(0, 3).join(".")` here? I believe it would look much cleaner.